packages/@aws-cdk/aws-lambda/lib/event-source-mapping.ts

```diff
@@ -101,6 +103,20 @@ export interface EventSourceMappingOptions {
    * @default - no topic
    */
   readonly kafkaTopic?: string;
+
+  /**
+   * The Secrets Manager secret ARN that stores your broker credentials.
+   *
+   * @default - no configuration
+   */
+  readonly kafkaSecretArn?: string
```

Comment:
Use `secrets.ISecret` as the type.

Reply:
Hm, doing that leads to "lerna ERR! ECYCLE @aws-cdk/aws-lambda -> @aws-cdk/aws-secretsmanager -> @aws-cdk/aws-lambda"

Reply:
Oh gosh. I'll need to look deeper under the hood to see what's going on and how we can resolve this, but taking the arn directly is not going to work.

Reply:
Solved by moving everything to the Kafka related event source properties.